parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -336,6 +340,18 @@ EOF
   systemctlEnableAndStart containerd || exit $ERR_SYSTEMCTL_START_FAIL
 }
 
+configureContainerdRegistryHost() {
+  # TODO(binxi): need to update for sovereign cloud.
+  CONTAINERD_CONFIG_REGISTRY_HOST_MCR="/etc/containerd/certs.d/mcr.microsoft.com/hosts.toml"
```

Comment:
why is this needed ?  for NI we expect users to be using private ACRs.  

why is it important to add `mcr.microsoft.com` for containerd ?

Reply:
With the containerd config registy host settings, the request to pull image from `mcr.microsoft.com` will pull image from the private ACR.